docs/debug_api.md

```diff
@@ -0,0 +1,165 @@
+Debug API
+=========
+
+
+The Debug API is a powerful HTTP API to make adjustments to the bridge at runtime, and is typically useful
+when administrating large bridges. Some functionality has moved to the [Admin Room](./admin_room.md)
+interface as sending commands over Matrix is preferred, however many powerful commands are exposed here.
+
+You can enable this feature in the config file:
+
+```yaml
+ircService:
+  # ...
+  debugApi:
+    enabled: true
+    port: 11100
+```
+
+**Note: The Debug API listens on `0.0.0.0` by default, so be careful to lock down access to this port.**
+
+To access the API over `curl`:
+
+```sh
+curl http://127.0.0.1:11100/inspectUsers?access_token=AS_TOKEN_FROM_REGISTRATION_FILE
+```
+
+## Endpoints
+
+<!-- no toc -->
```

Comment:
Did you build this manual Table of Content to ignore the `h4`s of each endpoint?

Reply:
Indeed, I felt having a list of each header was overkill. We only need a list of each endpoint.